packages/api-docs-builder/utils/createTypeScriptProject.ts

```diff
@@ -0,0 +1,81 @@
+import * as ts from 'typescript';
```

Comment:
This is a similar approach to what we are doing in `mui-x` and we will be able to create a wrapper around it for our needs.

We are creating on TypeScript tree per package and using it all along when we need to get information from TypeScript (instead of parsing file by file).